com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointPlanar.cs

```diff
@@ -166,16 +172,38 @@ protected override bool IsJointAxisDefined()
 #endif
         }
 
-        protected override void AdjustMovement(Joint joint)
+        protected override void SetAxisData(Vector3 axis)
         {
-            if (joint.axis == null || joint.axis.xyz == null)
+            axisofMotion = axis;
+            int motionAxis = -1;
+            for (int i = 0; i < 3; i++)
             {
-                joint.axis = new Joint.Axis(new double[] { 1, 0, 0 });
+                if (axisofMotion[i] > 0)
+                {
+                    motionAxis = i;
+                    break;
+                }
             }
-            axisofMotion = new Vector3((float)joint.axis.xyz[0], (float)joint.axis.xyz[1], (float)joint.axis.xyz[2]);
-            int motionAxis = joint.axis.AxisofMotion();
+            
             Quaternion motion = unityJoint.anchorRotation;
 
+            switch (motionAxis)
```

Comment:
is there a comment somewhere that states what 0, 1, 2 correspond to? x, y, z? any other order?